Resources/Textures/_Harmony/Mobs/Pets/gizard.rsi/meta.json

```
@@ -0,0 +1,21 @@
+﻿
+{
+  "version": 1,
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Lizard Sprite Made by Mithrandalf, Hat made by Flarbergaster",
```

This should say that you modified the sprites instead of just made the hat
```suggestion
  "copyright": "Lizard Sprite Made by Mithrandalf, modified by Flarbergaster to have a hat",
```